go.mod

```diff
@@ -4,8 +4,8 @@ go 1.12
 
 require (
 	github.com/blang/semver v3.6.1+incompatible // indirect
-	github.com/mattermost/mattermost-server v1.4.1-0.20191016162522-6597fdb40134 // Mattermost Server 5.16.0
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200525190709-91f010b8b7e8
```

Comment:
why this version?

Reply:
This was the latest master, we will switch it to 5.24

Reply:
`v5.24-rc1` is out: https://github.com/mattermost/mattermost-server/releases/tag/v5.24.0-rc1